### PR TITLE
Diep Custom

### PR DESCRIPTION
**Pull request by ghost**
Adds [Diepcustom](https://github.com/ABCxFF/diepcustom)
first time contributing to something other than mine!

### CODE REVIEW

File: diepcustom/diepcustom.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        "replace": "export const defaultAccessLevel: AccessLevel = AccessLevel.PublicAccess;",
+        "search": "export const defaultAccessLevel: AccessLevel = AccessLevel.BetaAccess;",
+        "type": "alterfile",
+        "regex": true
+      }
+    ],
+    "run": {
+      "stopCode": 2,
+      "command": "yarn run server",
+      "workingDirectory": "./",
```
Comment (LordRalex) [inline diepcustom/diepcustom.json]: Can leave blank

Comment (ghost) [reply diepcustom/diepcustom.json]: ok, thanks!

Comment (ghost) [reply diepcustom/diepcustom.json]: fixed

**Comment by ghost** [comment]
hey, on my main acc here. could someone close this? i wish to remake it

**Pull request closed without merging**